tests/Simpleue/Mocks/JobSpy.php

```diff
@@ -11,12 +11,18 @@
 class JobSpy implements Job {
 
     private $manageCounter;
+    private $quitCount;
+    private $testSignal;
 
     public function _construct() {
         $this->manageCounter = 0;
     }
 
     public function manage($job) {
+        if ($this->quitCount && (($this->manageCounter+1) === $this->quitCount)) {
```

Comment:
You can move the if after after incrementing the coutner ("$this->manageCounter++;"), so you can avoid doing it in the condition ("$this->manageCounter+1").